doc/source/management/data_at_rest_encryption.rst

```diff
@@ -396,6 +396,19 @@ tables should be encrypted again. If it is set to **1**, the encrypted table is
 re-encrypted on each key rotation. If it is set to **2**, the table is encrypted
 on every other key rotation.
       
+.. variable:: innodb_encrypt_online_alter_logs
+
+   :version 5.7.21-21: Implemented
+   :cli: ``--innodb-encrypt-online-alter-logs``
+   :dyn: Yes
+   :scope: Global
+   :vartype: Boolean
+   :default: OFF
+
+This variable simultaneously turns on the encryption of files used by InnoDB for
```

Comment:
- Also for full text search parallel sort.
- Please clarify where merge sort is used: index building
- For online DDL, only mention logs. No "temporary tables"

Reply:
While full text search and index building are relevant features that this variable is used for, merge sort and parallel sort (isn't this still a way to do the merge sort?) are implementation details. Are these implementation details essential for the description of this variable?

Reply:
Yes, these are things documented in MySQL manual and we'd like to answer the question whether they are encrypted

Reply:
Ready for review